source/River_Class.py

```diff
@@ -0,0 +1,42 @@
+from CardGames import *
+
+class River:
```

Comment:
I think this will benefit quite a bit down the road...

Reply:
Yeah, I wanted to make sure I had everything to get started